yaptide/routes/batch_routes.py

```diff
@@ -0,0 +1,152 @@
+from flask import request
+from flask_restful import Resource
+
+from marshmallow import Schema
+from marshmallow import fields
+
+from yaptide.routes.utils.decorators import requires_auth
+from yaptide.routes.utils.response_templates import yaptide_response, error_validation_response
+
+from yaptide.persistence.database import db
+from yaptide.persistence.models import UserModel, SimulationModel
+
+from yaptide.batch.batch_methods import submit_job, get_job, delete_job
+
+
+class JobsBatch(Resource):
+    """Class responsible for jobs via direct slurm connection"""
+
+    @staticmethod
+    @requires_auth(is_refresh=False)
+    def post(user: UserModel):
+        """
+        parametry:
+            sim_name: <string> (default - "")
+            sim_type: <string> (default - "shieldhit")
+            sim_data:
+                input_files: <list>
+                json projektu ui
+            slurm_params: (optional) - wyślij wiadomość, jeśli nie użyto
+                header: <string> (all parameters text block)
+                cmd_line_args: <słownik> | <None> (ma wygenerować stringa)
+
+        result not ok:
+            "Error message" (scenariusze):
+                error message od połączenia
+                error message batcha
+
+        result ok:
+            job_id
+        """
```

Comment:
```suggestion
```

move that info to issue description

Reply:
Ok

Reply:
https://github.com/yaptide/yaptide/issues/242#issuecomment-1430542060
Wrote it down here, consider resolving conversation